rules_web_testing_python/webtests/webtest_test.py

```diff
@@ -11,12 +11,11 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Tests for testing.web.webtest."""
+"""Tests for web.web_python.pythontests."""
```

Comment:
pls keep the original directory and this shouldn't need to change (nor user imports)